pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -285,8 +299,19 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 
 	for i := range config.Registries {
 		reg := &config.Registries[i]
-		// make sure Location and Prefix are valid
+		// make sure SubDomainPrefix, Location and Prefix are valid
 		var err error
+
+		// Prefer SubDomainPrefix over Location and Prefix
+		reg.SubDomainPrefix, err = parseSubDomainPrefix(reg.SubDomainPrefix)
+		if err != nil {
+			return err
+		}
+
+		if reg.SubDomainPrefix != "" {
+			return nil
```

Comment:
Why return here?  We still need to continue parsing the other fields and the remainder of `config.Registries` (we're in a loop).

Reply:
Ack, will take care of this..

Reply:
addressed in upcoming commit